CMakeLists.txt

```diff
@@ -567,23 +579,38 @@ else()
 endif()
 
 target_compile_features(torrent-rasterbar PUBLIC cxx_std_11)
-
-target_compile_definitions(torrent-rasterbar PUBLIC $<$<CONFIG:Debug>:TORRENT_DEBUG>)
-
-target_compile_definitions(torrent-rasterbar PRIVATE TORRENT_BUILDING_LIBRARY)
+target_compile_definitions(torrent-rasterbar
+	PUBLIC
+		$<$<CONFIG:Debug>:TORRENT_DEBUG>
+		$<$<CONFIG:Debug>:TORRENT_USE_ASSERTS>
+	PRIVATE
+		TORRENT_BUILDING_LIBRARY
+)
+target_link_libraries(torrent-rasterbar
+	PUBLIC
+		Threads::Threads
+)
 
 if (libiconv)
 	find_package(Iconv REQUIRED)
+	list(APPEND _package_dependencies "Iconv")
 	target_compile_definitions(torrent-rasterbar PUBLIC TORRENT_USE_ICONV)
-	target_include_directories(torrent-rasterbar SYSTEM PUBLIC ${ICONV_INCLUDE_DIR})
+	target_include_directories(torrent-rasterbar PUBLIC ${ICONV_INCLUDE_DIR})
+	target_link_libraries(torrent-rasterbar PRIVATE ${ICONV_LIBRARIES})
 endif (libiconv)
 
 if (encryption)
 	if(NOT DEFINED OPENSSL_INCLUDE_DIR OR NOT DEFINED OPENSSL_LIBRARIES)
 		FIND_PACKAGE(OpenSSL REQUIRED)
```

Comment:
is `FIND_PACKAGE` different from `find_package` in cmake? if not, I suppose this should be lower case too, like all other invocations.

Reply:
There is no case sensitivity here.